index.rst

```diff
@@ -558,11 +564,11 @@ Therefore, in cases where on-the-fly configuration changes are required, the fol
 
         await salobj.set_summary_state(ataos, salobj.State.ENABLED)
 
-    If an override configuration was modified/added, then you must specify it using the ``configurationOverride`` keyword
+    If an override configuration was modified/added, then you must specify it using the ``override`` keyword
 
     .. code-block:: python
 
-        await salobj.set_summary_state(ataos, salobj.State.ENABLED, configurationOverride='summit_constant_hex')
+        await salobj.set_summary_state(ataos, salobj.State.ENABLED, override='summit_constant_hex')
```

Comment:
This code example is out of date. The ATAOS does not provide this as an override.

Reply:
see comment below

Reply:
Note that I also added the file extension as it is now required.